homeworks/Andrii.Morozov_AndriiMorozov88/6-typesript/src/script_typescript.ts

```diff
@@ -0,0 +1,65 @@
+const letterBox = <HTMLDivElement>document.querySelector('[data-random-letter]');
+const scoreBox = <HTMLDivElement>document.querySelector('[data-score]');
+const scoreDif = <HTMLDivElement>document.querySelector('[data-score-different]');
```

Comment:
```suggestion
const letterBox = document.querySelector<HTMLDivElement>('[data-random-letter]');
const scoreBox = document.querySelector<HTMLDivElement>('[data-score]');
const scoreDif = document.querySelector<HTMLDivElement>('[data-score-different]');
```
Ти тут використав приведення типу до дів елемента, але `querySelector` підтримує дженерік типи